guides/common/modules/proc_logging-in-to-the-projectwebui-with-freeipa-credentials-in-mozilla-firefox.adoc

```diff
@@ -0,0 +1,35 @@
+[id="Using_{FreeIPA-context}_credentials_to_log_in_to_the_{ProjectWebUI-context}-with-Mozilla-Firefox_{context}"]
+= Logging in to the {ProjectWebUI} with {FreeIPA} credentials in Mozilla Firefox
```

Comment:
Vale would complain about "using" (it should be "by using"). This would make the heading longer -- not by much, but enough that I'd prefer to avoid it. No change.